.github/workflows/go.yml

```diff
@@ -19,6 +19,14 @@ jobs:
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #v3.5.2
       - name: Install Linters
         run: "curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.53.1"
+      - name: Use test and module cache
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 #v3.3.1
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
```

Comment:
Out of curiosity, how would we manually clear the cache if needed?

Reply:
You can manage caches under the actions tab -> Caches.  It's also possible via the API and `gh` CLI.

Reply:
![image](https://github.com/decred/dcrd/assets/2115102/07a6bf86-0484-43ac-8441-5cbb52b51dc6)

Reply:
Ah, great.  And as I understand it, `restore-keys` provides an ordered list of recent caches to use if there's no entry for the current `key`.  Good stuff, will use this in the other projects.

Reply:
Correct.

For reference, I do  know that there is still a bit of an issue for tests that involve fixtures.  Namely, when the actions check out (clone) the code, those fixtures get a new timestamp and the timestamp is part of the inputs hash that `go test` uses to determine if it can use the cached result.  It will probably need another step prior to executing the tests that does some magic on normalizing the timestamps somehow (probably based on the hash of the contents of the file) in order to handle those cases.

It's still useful as it is though.

Reply:
I've updated this to include a script that stabilizes the timestamps along with cache busting when the contents change.